### DISCUSSION TITLE
Is backing up the thumbnail cache necessary?

### DISCUSSION THREAD

**Discussion by GraysonCAdams** [photoprism/photoprism#1065] [Q&A] [answered]
I ask this because I have a recurring S3 backup of the cache/db (excluding JPEG files), but is that worthless without the associated thumbnails?

In my head, I imagine restoring everything to a library of broken previews, and if I were to re-index, would that not override any modifications to metadata I had made / be a complete purge of the current data?

**Comment by lastzero** [comment, marked as answer]
They can be recreated, so there is no backup needed. Just don't delete them intentionally as recreating thumbs takes a while.

**Reply by GraysonCAdams** [reply]
How do you recreate thumbs while maintaining your existing index and metadata?

**Reply by lastzero** [reply]
The indexer checks if they exists. If you deleted all you need to run a full rescan.
